asv_bench/benchmarks/io/csv.py

```diff
@@ -172,6 +172,26 @@ def time_head_of_multiindex(self):
         self.df_custom_index_then_head.to_csv(self.fname)
 
 
+class ToCSVSparse(BaseIO):
+
+    fname = "__test__.csv"
+
+    def setup(self):
+        from scipy import sparse as sc
+
+        vals = np.random.randint(0, 10, size=(10000, 1000))
+        keep = vals > 3
+        vals[keep] = 0
+        sparse_mtx = sc.coo_matrix(vals)
+        self.data = DataFrame.sparse.from_spmatrix(sparse_mtx)
+
+    def time_sparse_to_csv(self):
+        self.data.to_csv("sparse_pd.csv")
```

Comment:
Following the pattern of the benchmarks above, should this be `self.fname` instead?

Reply:
Yeah good call — updated in [6fa18f6](https://github.com/pandas-dev/pandas/pull/49066/commits/6fa18f642e3a242d3d716a322d6c51d11bb008c4)